lib/tasks/vacancies_to_update.yaml

```diff
@@ -212,3 +212,477 @@ We can offer:
+      starts_on: '01/09/2018'
+    -
+      slug: "teacher-of-ict-computing"
+      job_title: "Teacher of ICT"
+      job_description: "
+      <p>Prudhoe Community High School is a rapidly improving school; we have a new school and a real ambition to be one of the best schools in the north east. It is an exciting time to join this 13-19 Community High School.</p>
+      <p>We are offering an excellent opportunity for a suitably qualified ICT / Computing teacher to work within a high performing departmental team. We are really keen to look at candidates who can offer us something special.</p>
+      <p>This is a permanent variable contract (0.6 to 1.0 FTE) with a September 2018 start.</p>
+      <p>Application packs are available by contacting c.collings@pchs.org.uk  Closing date for applications is 12 noon on 8 May 2018. Applications received after this date will not be considered.</p>
+      <p>PCHS is committed to safeguarding the welfare of young people: an enhanced DBS check will be required</p>"
+      subject: "Computing"
+    -
+      slug: 'teacher-of-physical-education'
+      job_title: 'Teacher of Physical Education'
+      job_description: "
+      <p>St Aidan’s Catholic Academy is looking for a dynamic and committed individual to work in our PE department.You will be required to teach across the age range with a particular focus on the ability to teach examination PE including GCSE PE and Btec Sport.</p>
+      <p>We are a vibrant, diverse school community for 11-18 year old boys. Our academy has the privilege of teaching pupils who are inquisitive, engaged and thriving. Whilst the Academy has a relentless focus on high expectations and aspirations, we’re proud to provide a happy and nurturing environment, so that as well as maturing into well-rounded, confident young men, our pupils develop a passion for learning.</p>
+      <p>An Ofsted “Good School”, our most recent Ofsted report (January 2018) judged that “Pupils conduct themselves very well in lessons and around the school. They are confident, polite and courteous to one another and to staff. Pupils behave well and show positive attitudes to learning“ and that  “Leaders ensure that learning in subjects is effectively enhanced through the wide range of sporting and after-school activities on offer.“</p>
+      <p>The PE Department has a strong reputation both within the school and within the North East where we have a history of success in extra-curricular sport.</p>
+      <p>The Governing Body are committed to safeguarding and promoting the welfare of children and young people and vulnerable adults. The successful candidate will be subject to an enhanced disclosure through the Disclosure and Barring Service and additional recruitment checks.</p>
+      <p>NQTs are welcome to apply.</p>
+      <p>NB – Applications close Friday 11th May 2018.  Interviews will take place on Thursday 17th May.</p>
```

Comment:
Delete 'Applications close Friday 11th May 2018.' (indicated in box, and no specific time given so no added value)